src/building/house_service.c

```diff
@@ -118,6 +119,65 @@ void house_service_calculate_culture_aggregates(void)
                 }
             }
         }
+        
+        if (b->house_population > 0) {
+			if (b->data.house.theater) {
+				if (b->subtype.house_level >= HOUSE_SMALL_PALACE) {
+					b->data.house.theater_accessibility = 100;
+				} else {
+					b->data.house.theater_accessibility = calc_bound( (b->data.house.theater / b->house_population) * 100, 0, 100);
+				}
+			}
+			if (b->data.house.amphitheater_actor) {
+				if (b->subtype.house_level >= HOUSE_SMALL_PALACE) {
+					b->data.house.amphitheater_accessibility = 100;
+				} else {
+					b->data.house.amphitheater_accessibility = calc_bound( (b->data.house.amphitheater_actor / b->house_population) * 100, 0, 100);
+				}
+			}
+			if (b->data.house.colosseum_gladiator) {
+				if (b->subtype.house_level >= HOUSE_SMALL_PALACE) {
+					b->data.house.colosseum_accessibility = 100;
+				} else {
+					b->data.house.colosseum_accessibility = calc_bound( (b->data.house.colosseum_gladiator / b->house_population) * 100, 0, 100);
+				}
+			}
+			if (b->data.house.hippodrome) {
```

Comment:
I'm not sure if we should require all houses to have hippodrome access, as there can be only on in the entire city.

In order to give hippodrome access, we would have to spam the city with chariot makers, which kinda beats the point of removing culture bombs.

Check my comment regarding the `total_shows` on `ratings.c` for an alternative idea for hippodromes.